### DISCUSSION TITLE
Sno [Now Kart] & Github?

### DISCUSSION THREAD

**Discussion by bplewe** [koordinates/kart#376] [Q&A] [answered]
Sno looks very promising! Just wondering, is it possible to use sno with github or another web-hosted repository? All the documentation I can find describes setting up the sno repository in a local folder. I tried using my clone of the github repository as my sno repository, but that seems to defeat the advantages of sno (e.g. row-locking).

**Comment by rcoup** [comment, marked as answer]
You _can_ use any remote Git hosting including Github to host Sno repositories. It'll be ok for small datasets, though you'll see a lot of noise in the Web UI because the internals of Sno's data storage isn't human-readable in Github, and trying to do a Pull Request is a bad idea — branches need to be merged using `sno merge`. Medium/larger datasets (GB-range) will likely cause problems.

Koordinates has a preview of our Sno repository hosting available, please email us at support@koordinates.com with your Github username and we can get you set up.

We're intending to publish a configuration guide to help organisations that want to host Sno repositories on their own infrastructure, but in the interim [bare Git over SSH hosting](https://git-scm.com/book/en/v2/Git-on-the-Server-Getting-Git-on-a-Server) or a tool like [Gitolite](https://gitolite.com/gitolite/index.html) which doesn't try to put a web frontend on your repositories is a reasonable starting point.
